generate_ui.sh

```diff
@@ -0,0 +1,3 @@
+echo "Generating UI Files"
```

Comment:
you should add (shebang)[https://en.wikipedia.org/wiki/Shebang_(Unix)] 

also - you've added the `crowd_anki/user_files/Test/zzzNoteTypeExamples` and other test stuff again - please remove them and consider adding them to your local gitignore

Reply:
Shebang added :+1: 

I have added test things again? :upside_down_face: I was so careful :sob: but how come I cannot find them? :thinking: Both this changeset list lacks them (in my view), and my branch directory is empty: https://github.com/ll-in-anki/CrowdAnki/tree/sortnotesconfigoption/crowd_anki/user_files :thinking:

Reply:
mm, not sure 😛 they don't seem to be there anymore 😅